Update README (#1792)
LiskHQ/lisk-mobile: README.md

> Lisk Mobile is a cross-platform application written in React Native and primarily built for iOS and Android. It provides the users with all the functionality they need to send and receive LSK tokens, as well as review the activity history of their or any other account on the Lisk blockchain.

Reviewer: ```suggestion
Lisk Mobile is a cross-platform application written in React Native and primarily built for iOS and Android. It provides all the functionalities required that allow the users to send and receive LSK tokens, coupled with the ability to access the activity history of any Lisk blockchain account. 
```

### iOS

> - Install the latest version of [Xcode](https://apps.apple.com/ng/app/xcode/id497799835?mt=12) to use simulator

Reviewer: ```suggestion
- Install the latest version of [Xcode](https://apps.apple.com/ng/app/xcode/id497799835?mt=12) to use the simulator.
```

### Android

> - Install [Android Studio](https://developer.android.com/studio/index.html), ensure the below listed options are checked and installed:

Reviewer: ```suggestion
- Install [Android Studio](https://developer.android.com/studio/index.html), and ensure the options listed below are also checked and installed:
```

> More details [here](https://armen-mkrtchian.medium.com/run-cocoapods-on-apple-silicon-and-macos-big-sur-developer-transition-kit-b62acffc1387).

Reviewer: ```suggestion
More details can be found [here](https://armen-mkrtchian.medium.com/run-cocoapods-on-apple-silicon-and-macos-big-sur-developer-transition-kit-b62acffc1387).
```

## Development

> You can run the project in Xcode and use iOS simulators, or alternatively use Android simulators. There are several options to set up your Android development environment. Please read [React Native docs](https://facebook.github.io/react-native/docs/getting-started.html) for more information.

Reviewer: ```suggestion
You can run the project in Xcode and use iOS simulators, or alternatively use Android simulators. There are several options to set up your Android development environment. Please read the [React Native docs](https://facebook.github.io/react-native/docs/getting-started.html) for more information.
```

> There is a standalone app for debugging React Native. It has React and Redux debugger enabled by default. Please read [React Native Debugger](https://github.com/jhen0409/react-native-debugger) for more information.

Reviewer: ```suggestion
There is a standalone app for debugging React Native which has a React and Redux debugger enabled by default. Please read the [React Native Debugger](https://github.com/jhen0409/react-native-debugger) for more information.
```

/Users/***/Library/Developer/Xcode/DerivedData/Lisk-***/Build/Products/Debug-iphonesimulator/lottie-ios/Lottie.modulemap
```

> Follow these steps to solve it:

Reviewer: ```suggestion
Follow these steps below which should resolve any errors:
```

> 1. Add the following to the end of your Podfile (especially M1 users)

Reviewer: ```suggestion
1. Add the following to the end of your Podfile (especially M1 users).
```
